src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -4,10 +4,30 @@
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
-public class Bulldozer extends Machine {
+
+public class Bulldozer extends Machine implements Workable {
```

Comment:
why implements Workable ???
```suggestion
public class Bulldozer extends Machine {
```